docs/reference/Admin_Actions.md

```diff
@@ -203,3 +203,1432 @@ Modified some packages data on the backend database, due to incompatibility with
+* synders-toolkit
+* syntagma-ui
+* syntax-obsidian
+* syntax-tree
+* system-font-snippets
+* t2
+* t808-syntax
+* tabclick
+* tailwind-dark-syntax
+* tailwind-light-syntax
+* take-a-note-syntax
+* talk-like-a-pirate
+* tapplint
+* tarocchi-coffee
+* teran-ui
+* termal-syntax
+* termal-ui
+* tern-j-s
+* ternjs
+* terraform-lookup
+* test-package-1
+* test-rename-1
+* test-status
+* testpackage1
+* testrepo
+* the-grid
+* thf-snippets
+* thinkphp5-snippets
+* time-status
+* timekeeper
+* tochelet-ui
+* todo-panel
+* toma-syntax
+* toma-theme
+* tomorrow-night-blue-syntax
+* tomorrow-night-bright-syntax
+* tomorrow-night-saturated-syntax
+* tomorrow-syntax
+* tool-bar-basic
+* tool-bar-noderat
+* tool-bar-rawn
+* toolbar-cmake-shell
+* toothpaste_2_syntax
+* translation-helper
+* transmit
+* travis-ci-status
+* tree-lines
+* tree-view-bundler
+* tree-view-no-drag
+* tree-view-remote
+* treeview-file-highlight
+* trendy-dark-syntax
+* trendy-light-ui
+* tron-legacy-syntax-cp
+* tron-legacy-syntax
+* tron-legacy-ui
+* true-emoticons
+* try-catch-wrapper
+* ts-angular-styleguide
+* tsubaki-syntax
+* tutors-lessons-syntax
+* twilight-textmate-syntax-refined
+* two-dark-ui
+* two-light-ui
+* two-syntax
+* typingeffect
+* ui-minory
+* uikit-jade
+* ukuhlanekezela
+* ultra-snazzy-syntax
+* ultra-snazzy
+* ultra-syntax
+* ultra
+* umsl-toolbar
+* underscore
+* underscorecase
+* underscorejs-snippets-no-semicolon
+* unis
+* untime-syntax
+* update-package-dependencies-elastic
+* urlrequest
+* userlite-link
+* userlite-snippets
+* userlite-utilities
+* v-code
+* vacuous-syntax
+* valhalla
+* vemv-autocomplete
+* venus-runner
+* veryspecial-syntax
+* vibranium-syntax
+* vim-like-syntax
+* vim-mode-plus-keymaps-for-git
+* vim-mode-plus-keymaps-for-tree-view
+* voz-living-dev-tool
+* vs2015-dark-syntax
+* vsc-dark-syntax
+* vue-autocomplete
+* vue-snippets
+* vue2-autocomplete
+* vuetify-atom
+* w-atom-snippet
+* wakanda-syntax
+* warbp-3r-syntax
+* warmth-syntax
+* web-developer-light-syntax
+* webbox-color
+* willyelm-test2
+* willyelm-test4
+* wilmersdorf-syntax
+* wl-jshinter
+* words
+* wtf-preview
+* wynnscript-atom
+* x-tools
+* xavil-syntax
+* xcode-bracket-matcher
+* xcode-midnight-dark-syntax
+* xiaolais-atom-snippets
+* xkcd-comics
+* xmc-workspace
+* xolotl
+* yjy-syntax
+* yoceanic-next-syntax
+* yolo-wallhack-syntax
+* yongky-atom-theme
+* yosemite-ui
+* youngjuning-snippets
+* yuma-sintax
+* yuno-syntax
+* z3editor
+* zan-language-ruby
+* zaraa
+* zc-dark-syntax
+* zen-monk
+* zenburnesque
+* zenchat-snippets
+* ziv-syntax
+
+### Name is Banned
+
+Due to the severe amount of spam the Atom Package Registry started to see in the last days, a ban list of names was created to prevent spam or malicious package's from being migrated over. Luckily the majority of spam package's were created in the following days after the initial migration was made, but still some package's had to be prevented from being repacked, which are listed below:
```

Comment:
```suggestion
Due to the severe amount of spam the Atom Package Registry started to see in the last days, a ban list of names was created to prevent spam or malicious packages from being migrated over. Luckily the majority of spam packages were created in the following days after the initial migration was made, but still some packages had to be prevented from being repacked, which are listed below:
```